client/conferences.js

```diff
@@ -0,0 +1,48 @@
+// This file is built with parcel-bundler locally.
+// Dist is static/js
+import moment from 'moment'
+import lodash from 'lodash'
+
+/*  eslint-env jquery */
+
+function onLoadEvent () {
+  console.log('loaded')
```

Comment:
all good so far, if possible please remove this `console.log`

Reply:
Of course. Again, I need to fix the tools. This can be a serious performance problem, also.

Reply:
"serious"